mitosheet/src/mito/utils/actions.tsx

```diff
@@ -714,6 +787,61 @@ export const getActions = (
             searchTerms: ['filter', 'remove', 'delete'],
             tooltip: "Filter this dataframe based on the data in a column."
         },
+
+        [ActionEnum.FilterToCellValue]: {
+            type: 'build-time',
+            staticType: ActionEnum.FilterToCellValue,
+            titleContextMenu: 'Filter to Cell Value',
+            iconContextMenu: FilterIcon,
+            longTitle: 'Filter column',
+            actionFunction: async () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                const isNaN = cellValue === undefined || isValueNone(cellValue)
+                let condition = 'string_exactly';
+                if (columnDtype === 'number') {
+                    condition = 'number_exactly';
+                } else if (columnDtype === 'boolean') {
+                    condition = cellValue ? 'boolean_is_true' : 'boolean_is_false';
+                } else if (columnDtype === 'datetime') {
+                    condition = 'datetime_exactly';
+                }
+                if (isNaN) {
+                    condition = 'empty'
```

Comment:
Wanna refactor this into a util like the exclusive one? I think it's the inclusive version?